src/Compute.js

```diff
@@ -0,0 +1,261 @@
+import React, { useState } from 'react'
+
+import { asset } from './asset'
+import { algoAsset, createComputeService, rawAlgoMeta } from './asset-compute'
+
+export default function Compute({ ocean, web3 }) {
+  const [ddoAssetId, setDdoAssetId] = useState('')
+  const [jobStatus, setJobStatus] = useState('')
+  const [jobId, setJobId] = useState('')
+  const [agreementId, setAgreementId] = useState('')
+  const [ddoAlgorithmId, setDdoAlgorithmId] = useState('')
+  const [isAlgoInputVisible, setIsAlgoInputVisible] = useState('')
+  const [textRawAlgo, setTextRawAlgo] = useState('')
+  const [publishLogState, setPublishLogState] = useState(false)
+  const [publishOutputState, setPublishOutputState] = useState(false)
+
+  // publish a dataset and an algorithm
+  async function publish() {
+    try {
+      const accounts = await ocean.accounts.list()
+      console.log('Publishing asset.')
+
+      const service = await createComputeService(
+        ocean,
+        accounts[0],
+        '0',
+        '2020-03-10T10:00:00Z'
+      )
+      console.log(service)
+      const ddoAssetNew = await ocean.assets.create(asset, accounts[0], [
+        service
+      ])
+      console.log('Asset successfully submitted.')
+      console.log(ddoAssetNew)
+      // keep track of this registered asset for consumption later on
+      setDdoAssetId(ddoAssetNew.id)
+      alert('Asset successfully submitted.')
+    } catch (error) {
+      console.error(error.message)
+    }
+  }
+
+  async function publishalgo() {
+    try {
+      const accounts = await ocean.accounts.list()
+      console.log('Publishing algo.')
+
+      const ddoAlgorithmNew = await ocean.assets.create(algoAsset, accounts[0])
+      console.log(ddoAlgorithmNew)
+      console.log('Algo asset successfully submitted.')
+      // keep track of this registered asset for consumption later on
+      setDdoAlgorithmId(ddoAlgorithmNew.id)
+      alert('Algorithm successfully submitted.')
+    } catch (error) {
+      console.error(error.message)
+    }
+  }
+
+  async function startCompute(algorithmId, algorithmMeta) {
+    try {
+      const accounts = await ocean.accounts.list()
+      const ComputeOutput = {
```

Comment:
to keep it consistent, only React components start with uppercase letter. This should be `computeOutput`